### ISSUE TITLE
CSS file that's imported from JS file is not being included in the manifest

### ISSUE THREAD

**Issue by dudo** [autifyhq/parcel-reporter-bundle-manifest#101]
Hey there! I can't seem to get css to register. I've deleted dist and .parcel-cache, and rebuilt. Any other ideas?

![image](https://user-images.githubusercontent.com/2963800/97382101-d77ee000-1887-11eb-977d-817dd7ad18ea.png)

**Comment by lukyth** [comment]
Is it possible to share the setup that reproduce this?

**Comment by lukyth** [comment]
@dudo Is it possible to share the setup that reproduce this?

**Comment by dudo** [comment]
```js
// package.json
{
  "name": "browse_app",
  "version": "0.1.0",
  "private": true,
  "description": "",
  "scripts": {
    "start": "parcel serve ./src/index.html -p 1264 --hmr-port 1265",
    "build": "parcel build ./src/index.html"
  },
  "dependencies": {
    "@babel/runtime": "^7.12.1",
    "body-scroll-lock": "^3.1.5",
    "framer-motion": "^2.9.4",
    "json-api-serializer": "^2.5.0",
    "react": "^17.0.0",
    "react-dom": "^17.0.0",
    "react-router-dom": "^5.2.0"
  },
  "devDependencies": {
    "@babel/core": "^7.12.3",
    "@babel/plugin-transform-react-jsx": "^7.12.1",
    "@babel/plugin-transform-runtime": "^7.12.1",
    "@babel/preset-env": "^7.12.1",
    "babel-eslint": "^10.1.0",
    "eslint": "^7.12.1",
    "eslint-plugin-react": "^7.21.5",
    "eslint-plugin-react-hooks": "^4.2.0",
    "parcel": "^2.0.0-nightly.433",
    "parcel-reporter-bundle-manifest": "^0.3.1"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```
```js
// .babelrc
{
  "plugins": [
    [
      "@babel/plugin-transform-runtime",
      {
        "regenerator": true
      }
    ],
    [
      "@babel/plugin-transform-react-jsx",
      {
        "runtime": "automatic"
      }
    ]
  ]
}
```
```js
// .parcelrc
{
  "extends": "@parcel/config-default",
  "reporters": ["...", "parcel-reporter-bundle-manifest"]
}
```

Setup was pretty straight forward... thanks for taking a look =)

**Comment by lukyth** [comment]
@dudo And the CSS file is being imported from `./src/index.html` with something like `<link rel="stylesheet" href="./styles.css" />` right? Would it be possible to share a minimal setup that could reproduce this via [CodeSandbox](https://codesandbox.io/)?

**Comment by dudo** [comment]
@lukyth that was it! I was importing the main css file from my js entrypoint. Removing that in favor of including the link meta tag properly includes `index.css`. 

Thanks!!

**Comment by lukyth** [comment]
I see. Thanks for the confirmation! I think it should include that CSS file in the generated manifest as well so I'm gonna reopen this issue until it's resolved. And thanks for reporting btw! 👍

**Comment by lukyth** [comment]
@dudo Which version of Parcel 2/this plugin/Node.js that you are using? With those info, I might be able to reproduce the issue and help you with it.

I tried to build with a CSS file imported from JS file, and I could see the CSS file in the manifest as well:
<img width="480" alt="Screen Shot 2020-11-02 at 1 06 39" src="https://user-images.githubusercontent.com/7040242/97808260-376eef80-1ca9-11eb-9f76-456aff0f10a4.png">

The setup I tested with is at https://github.com/autifyhq/parcel-reporter-bundle-manifest/tree/main/tests/fixtures/parcel-beta, with the JS import CSS added in https://github.com/autifyhq/parcel-reporter-bundle-manifest/commit/34a06905638c7bda3242945251a60501399b6ebc.

**Comment by dudo** [comment]
```js
"dependencies": {
    "@babel/runtime": "^7.12.1",
    "react": "^17.0.0",
    "react-dom": "^17.0.0",
    "react-router-dom": "^5.2.0"
  },
  "devDependencies": {
    "@babel/core": "^7.12.3",
    "@babel/plugin-transform-react-jsx": "^7.12.1",
    "@babel/plugin-transform-runtime": "^7.12.1",
    "@babel/preset-env": "^7.12.1",
    "babel-eslint": "^10.1.0",
    "eslint": "^7.12.1",
    "eslint-plugin-react": "^7.21.5",
    "eslint-plugin-react-hooks": "^4.2.0",
    "parcel": "^2.0.0-nightly.433",
    "parcel-reporter-bundle-manifest": "^0.3.1"
  },
```

**Comment by lukyth** [comment]
@dudo This should be fixed in [v0.3.2](https://www.npmjs.com/package/parcel-reporter-bundle-manifest/v/0.3.2) now. Let me know if you still find any problem! 😃
